### PR TITLE
Add nssdb files for EL8

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: manifests/ssltools/nssdb.pp [hunk 1]
```diff
@@ -6,7 +6,15 @@
   Exec { logoutput => 'on_failure' }
 
   $nss_db_password_file   = "${nss_db_dir}/nss_db_password-file"
-  $nssdb_files            = ["${nss_db_dir}/cert8.db", "${nss_db_dir}/key3.db", "${nss_db_dir}/secmod.db"]
+
+  case $operatingsystemrelease {
```
Comment (ekohl) [inline manifests/ssltools/nssdb.pp]: You could also use `$operatingsystemmajrelease` which is only the major version. Nowadays we should also use `$facts['operatingsystemmajrelease']`.

Comment (ehelms) [reply manifests/ssltools/nssdb.pp]: Updated

File: manifests/ssltools/nssdb.pp [hunk 2]
```diff
@@ -6,7 +6,15 @@
   Exec { logoutput => 'on_failure' }
 
   $nss_db_password_file   = "${nss_db_dir}/nss_db_password-file"
-  $nssdb_files            = ["${nss_db_dir}/cert8.db", "${nss_db_dir}/key3.db", "${nss_db_dir}/secmod.db"]
+
+  case $facts['operatingsystemmajrelease'] {
```
Comment (ekohl) [inline manifests/ssltools/nssdb.pp]: I think I agree with @evgeni that we should be future compatible. How about:
```puppet
$nssdb_files = $facts['operatingsystemmajrelease'] ? {
  '7'     => ["${nss_db_dir}/cert8.db", "${nss_db_dir}/key3.db", "${nss_db_dir}/secmod.db"],
  default => ["${nss_db_dir}/cert9.db", "${nss_db_dir}/key4.db", "${nss_db_dir}/pkcs11.txt"],
}
```

**Comment by ehelms** [comment]
Updated to change the default

**Pull request merged**